src/modules/content/api.ts

```diff
@@ -0,0 +1,14 @@
+import api from "api";
+import { ContentResponse } from "./types";
+
+export const extendedApi = api.injectEndpoints({
+  endpoints: (build) => ({
+    getContent: build.query<ContentResponse, void>({
+      query: () => "v1/content",
+    }),
+  }),
+});
```

Comment:
It looks like this endpoint isn't working on the back-end yet, but I think it's probably fine to keep this code here for now.